### PR TITLE
Add new fire method for publishing

### PR DESCRIPTION
**Pull request by jzucker2**

### CODE REVIEW

File: PubNub/Core/PubNub+Publish.h [hunk 1]
```diff
@@ -829,6 +829,21 @@ self.client = [PubNub clientWithConfiguration:configuration];
          compressed:(BOOL)compressed withMetadata:(nullable NSDictionary<NSString *, id> *)metadata
          completion:(nullable PNPublishCompletionBlock)block;
 
+#pragma mark - Fire (no rep)
+
+/**
+ @since 4.4.0
+ */
+- (void)fire:(nullable id)message toChannel:(NSString *)channel completion:(nullable PNPublishCompletionBlock)block;
```
Comment (parfeon) [inline PubNub/Core/PubNub+Publish.h]: `fire:(nullable id)message` - when there is no other potential sources of message payload (like mobile push payload) there shouldn't be _nullable_ in signature.

Comment (jzucker2) [inline PubNub/Core/PubNub+Publish.h]: Good point

File: PubNub/Core/PubNub+Publish.m [hunk 2]
```diff
@@ -413,6 +465,10 @@ - (PNRequestParameters *)requestParametersForMessage:(NSString *)message toChann
         [parameters addQueryParameter:[PNString percentEscapedString:metadata] forFieldName:@"meta"];
     }
     
+    if (!shouldReplicate) {
+        [parameters addQueryParameter:@"1" forFieldName:@"no-rep"];
```
Comment (MaxPresman) [inline PubNub/Core/PubNub+Publish.m]: i just double checked the docs and it's `norep`, I made a mistake in the pivotal here is the same in [java](https://github.com/pubnub/java/blob/768b539a728abfb4e273c29c3fcb31260aa8f859/src/main/java/com/pubnub/api/endpoints/pubsub/Publish.java#L96)

Comment (jzucker2) [inline PubNub/Core/PubNub+Publish.m]: That's an easy fix

**Comment by parfeon** [comment]
Already implemented.

**Pull request closed without merging**